sci-libs/openlibm/openlibm-0.7.5.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="High quality system independent, portable, open source libm implementation"
+DESCRIPTION="OpenLibm is an effort to have a high quality, portable, \
+standalone C mathematical library (libm). It can be used standalone in \
+applications and programming language implementations.
+
+The project was born out of a need to have a good libm for the Julia \
+programming langage that worked consistently across compilers and operating \
+systems, and in 32-bit and 64-bit environments."
+HOMEPAGE="https://openlibm.org"
+COPYRIGHT="2011-14 The Julia Project. \
+	2008 Stephen L. Moshier \
+	1992-2011 The FreeBSD Project. \
+	1993 by Sun Microsystems, Inc. \
+	1997, 1999 Free Software Foundation, Inc."
+LICENSE="OpenLibm"
```

Comment:
MIT, BSD (2-clause), ICS and GNU LGPL v2.1 should be added here too

https://github.com/JuliaMath/openlibm/blob/master/LICENSE.md

Reply:
Seeing they are mentioned in the OpenLibm maybe not, @korli any thoughts (I see other ports at repology mentioning them)

Reply:
This is a custom mix of licenses, therefore "OpenLibm" as a placeholder. The document outlines various caveats there, which will be lost if just replacing it with other generic license templates.

Reply:
Combining different licenses doesn't make any difference. Openlibm uses multiple licenses for different parts, they should be detailed.

Reply:
Fixed